index.js

```diff
@@ -144,6 +149,31 @@ class HLSSpliceVod {
     });
   }
 
+  insertBumper(bumperMasterManifestUri, _injectBumperMasterManifest, _injectBumperMediaManifest) {
+    return new Promise((resolve, reject) => {
+      this._parseAdMasterManifest(bumperMasterManifestUri, _injectBumperMasterManifest, _injectBumperMediaManifest)
+      .then(bumper => {
+        const bandwidths = Object.keys(this.playlists);
+        for (let b = 0; b < bandwidths.length; b++) {
+          const bw = bandwidths[b];
+
+          const bumperPlaylist = bumper.playlist[findNearestBw(bw, Object.keys(bumper.playlist))];
+          const bumperLength = bumperPlaylist.items.PlaylistItem.length;
+          let i = 0;
```

Comment:
Does `i` get a value different than 0 anywhere?

Reply:
No actually not as a bumper should always be placed first. I kept it for clarity as it is the same case as for inserting an ad with offset = 0 which would yield an i = 0. Maybe it is more logical to change this...

Reply:
I have changed this now